compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -2193,21 +2192,13 @@ pub fn encode_metadata(tcx: TyCtxt<'_>, path: &Path) {
     // there's no need to do dep-graph tracking for any of it.
     tcx.dep_graph.assert_ignored();
 
-    join(
```

Comment:
I would strongly prefer this be changed to use `disabled::join` so it could be easily reverted.